pkg/chartutil/create.go

```diff
@@ -59,6 +61,18 @@ service:
   type: ClusterIP
   externalPort: 80
   internalPort: 80
+ingress:
+  enabled: false
+  # Used to create Ingress record (should used with service.type: ClusterIP).
+  # hostname: chart-example.local
```

Comment:
@thomastaylor312 OK I have updated PR per discussion so far. Let me know if other changes are needed (I assume there's a lot to discuss, but perhaps we can do it iteratively?